bittensor/_cli/cli_impl.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, config: 'bittensor.Config' ):
                 config (:obj:`bittensor.Config`, `required`): 
                     bittensor.cli.config()
         """
-        if not config.no_version_checking:
+        if config.get('no_version_checking') != None and not config.no_version_checking:
```

Comment:
Could be `if not config.get('no_version_checking'):`   

because None is falsey and indicates False here also

Reply:
please, add the code suggestion and we add it right away :)